build.rs

```diff
@@ -27,25 +27,25 @@ fn main() {
     let tags: DriverManifest = serde_json::from_str(drivers_json.as_str())
         .expect("Failed to parse drivers.json");
 
-    let out = std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap());
+    let drivers_dir = dbg!(std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap()).join("drivers"));
```

Comment:
`dbg!` should be removed?

Reply:
Yes, probably, although all this output is generally hidden anyway (see existing `dbg!` on line 15). I'm happy to remove it though.